src/locales/cs.ts

```diff
@@ -0,0 +1,796 @@
+    immaculate_conception_of_mary_patroness_of_the_usa: 'Panny Marie, počaté bez poskvrny prvotního hříchu, patronky USA',
+    peter_chanel_priest_patron_of_oceania: 'Sv. Petra Chanela, prvního mučedníka Oceánie a patrona Oceánie',
+    john_brenner_priest: 'Bl. Jana Brennera, kněze a mučedníka',
+    paul_of_thebes_hermit: 'Bl. Pavla z Téb, poustevníka',
+    eusebius_of_esztergom_priest: 'Bl. Eusebia Ostrihomského, kněze',
+    pavel_peter_gojdic_bishop: 'Bl. Pavla Petra Gojdiče, biskupa a mučedníka',
+    szilard_bogdanffy_bishop: 'Bl. Szilárda Bogdánffyho, biskupa a mučedníka',
+    ladislaus_batthyany_strattmann: 'Bl. Ladislava Batthyány-Strattmanna',
+    zoltan_lajos_meszlenyi_bishop: 'Bl. Zoltána Ludvíka Meszlényiho, biskupa a mučedníka',
+    ceferino_gimenez_malla_martyr: 'Bl. Zeffirína Giménez Malla, mučedníka',
+    john_scheffler_bishop: 'Bl. Jana Schefflera, biskupa a mučedníka',
+    our_lady_of_sorrows_patroness_of_slovakia: 'Panny Marie Bolestné, patronky Slovenska',
+    george_of_lydda_martyr_patron_of_england: 'Sv. Jiří, mučedníka a patrona Anglie',
+    benedict_of_nursia_abbot_patron_of_europe: 'Sv. Benedikta, opata, patrona Evropy',
+    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_copatrons_of_europe: 'Sv. Cyrila, mnicha, a Metoděje, biskupa, spolupatronů Evropy',
+    cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_slavic_missionaries: 'Sv. Cyrila a Metoděje, slovanských věrozvěstů',
+    teresa_benedicta_of_the_cross_stein_virgin_copatroness_of_europe: 'Sv. Terezie Benedikty od Kříže, panny, mučednice a spolupatronky Evropy',
+    andrew_apostle_patron_of_scotland: 'Sv. Ondřeje, apoštola a patrona Skotska',
+    david_of_wales_bishop_patron_of_wales: 'Sv. Davida, biskupa a patrona Walesu',
+    bridget_of_sweden_religious_copatroness_of_europe: 'Sv. Brigity, řeholnice a spolupatronky Evropy',
+    catherine_of_siena_virgin_copatroness_of_europe: 'Sv. Kateřiny Sienské, panny, učitelky církve a spolupatronky Evropy',
+    catherine_of_siena_virgin_copatroness_of_italy_and_europe: 'Sv. Kateřiny Sienské, panny, učitelky církve a spolupatronky Itálie a Evropy',
+    patrick_of_ireland_bishop_patron_of_ireland: 'Sv. Patrika, biskupa a patrona Irska',
+    columba_of_iona_abbot_copatron_of_ireland: 'Sv. Kolumby, Abbot, Missionary and Copatron of Ireland',
+    our_lady_of_guadalupe_patroness_of_the_americas: 'Panny Marie Guadalupské, patronky obou Amerik',
+    francis_of_assisi_patron_of_italy: 'Sv. Františka z Assisi, patrona Itálie',
+    adalbert_of_prague_bishop_patron_of_poland: 'Sv. Vojtěcha, biskupa, mučedníka a patrona Polska',
+    stanislaus_of_szczepanow_bishop_patron_of_poland: 'Sv. Stanislava, biskupa, mučedníka a patrona Polska',
+    wenceslaus_i_of_bohemia_martyr_patron_of_czech_nation: 'Sv. Václava, mučedníka, hlavního patrona českého národa',  // TODO: Add this to all other locales + czechRepublic calendar
```

Comment:
Hey guys!

I need some proofing of you (just a single string :smiley:). Most of these should be correct _per se_.

Notes:
- @igneus: I’m quite confident the Latin translation is not grammatically correct. :wink:
- @igneus: I dropped the `hlavního` word because of how romcal works / will work, at least for now. Looking forward to your findings on the patronage matter in the Czech liturgical usage though. :smiley:

Assignments:
- @CodingAleCR: Spanish;
- @emagnier: French;
- @igneus: Latin;
- @gtortone: Italian;
- @jarosz: Polish;
- @HenriqueDerosa: Brazilian Portuguese.


```js
// Original: Czech
wenceslaus_i_of_bohemia_martyr_patron_of_czech_nation:
  'Sv. Václava, mučedníka a patrona českého národa',

en : 'Saint Wenceslaus, Martyr and patron of the Czech nation',
es : 'San Wenceslao de Bohemia, Mártir y Patrono de la nación checa',  // TODO: proof; also add a key for the celebration without patronage title
fr : 'Saint Venceslas, Martyr et Patron de la nation tchèque (✝ 929)',  // TODO: proof
it : 'San Venceslao, martire e patrono della nazione ceca',  // TODO: proof
la : 'S. Wenceslai, martyris et patroni de gente Bohemica',  // TODO: proof
pl : 'Św. Wacława, męczennika i patrona narodu czeskiego',  // TODO: proof
pt-br : 'São Venceslau, mártir e patrono da nação tcheca',  // TODO: proof
sk : 'Svätého Václava, mučeníka a patróna českého národa',
```

**PS—Thanks in advance! :wink:**

Reply:
`fr : 'Saint Venceslas, Martyr et Patron de la Tchéquie (✝ 929)'`

Reply:
@emagnier, thanks for your comment, however, your localisation is not quite correct. I had a conversation about this with @igneus (a native Czech). I too thought that `the Czech nation` equals the Czech Republic / Czechia, however, it is not so. The difference is subtle, but it does exist.

Refer please to this [homily of Benedict XVI](https://www.vatican.va/content/benedict-xvi/fr/homilies/2009/documents/hf_ben-xvi_hom_20090928_san-venceslao.html). In the heading, you can find `Saint Venceslas, patron de la nation`. I simply added the `tchèque` word to distinguish of which nation he is a patron.

Reply:
Thanks for the clarifications. We are less used to hearing this in this form in French, but I understand the difference. So keep what you have suggested initially.

Reply:
> We are less used to hearing this in this form in French

So am I and I think most Slovaks too. :smiley:

Anyway, thanks for the proofing. :wink:

Reply:
Hi, the Spanish translation looks good to me. Cheers!

Reply:
I suggest we should not wait any longer for others to approve my translations. I am fairly confident that they are all more or less correct.